ldaptor/test/test_distinguishedname.py

```diff
@@ -315,29 +315,38 @@ def testMalformed(self):
 
 class LDAPDistinguishedName_Prettify(unittest.TestCase):
     def testPrettifySpaces(self):
-        """str(DistinguishedName(...)) prettifies the DN by removing extra whitespace."""
+        """DistinguishedName(...).toWire() prettifies the DN by removing extra whitespace."""
         d=dn.DistinguishedName('cn=foo, o=bar,  c=us')
-        assert str(d) == 'cn=foo,o=bar,c=us'
+        assert d.toWire() == b'cn=foo,o=bar,c=us'
 
 class DistinguishedName_Init(unittest.TestCase):
-    def testString(self):
+    def testToWire(self):
         d=dn.DistinguishedName('dc=example,dc=com')
-        self.assertEqual(str(d), 'dc=example,dc=com')
+        self.assertEqual(d.toWire(), b'dc=example,dc=com')
 
     def testDN(self):
         proto=dn.DistinguishedName('dc=example,dc=com')
         d=dn.DistinguishedName(proto)
-        self.assertEqual(str(d), 'dc=example,dc=com')
+        self.assertEqual(d.toWire(), b'dc=example,dc=com')
+
+    def testEqualToByteString(self):
+        d = dn.DistinguishedName('dc=example,dc=com')
+        assert d == b'dc=example,dc=com'
```

Comment:
I know that previous code was using `assert` but I think that using self.assertEqual() will get a better output in the case of errors and will make future work easier

Reply:
You are right. I was afraid that it will not work this way but didn't check it. Fixed!